docs/subnets/evm-customization/geth-coreth-subnetevm-precopmpileevm.md

```diff
@@ -0,0 +1,30 @@
+# Geth, Coreth, Subnet-EVM and Precompile-EVM
```

Comment:
```suggestion
# Exemplary Virtual Machines
```
So that we can easily add more in the future

Reply:
I wrote this article to explain the relationships between the many different EVM-forks on Avalanche. I would not add other non-EVM VMs (timestamp etc.) in there, since this article is to create clarity around the EVMs. If you don't like  the current title, we could change it to "EMV-based VMs on Avalanche" or something